source/scripting/Entity.cs

```diff
@@ -37,37 +37,93 @@ public IntPtr MemoryAddress
 		}
 
 		/// <summary>
-		/// Gets or sets the health of this <see cref="Entity"/>.
+		/// Gets or sets the health of this <see cref="Entity"/> in int.
 		/// </summary>
 		/// <value>
-		/// The health from 0 - 100.
+		/// The health in int.
```

Comment:
Dont like this at all, can you keep the from 0 - 100, but put as an integer after